web_listview_range_select/readme/CONTRIBUTORS.rst

```diff
@@ -1,3 +1,4 @@
+* Nilesh Sheliya <nilesh@synodica.com>
```

Comment:
Add your name under the previous contributors.

Reply:
Ok

Reply:
Added name under the previous contributors.